python/python/glide/async_commands/core.py

```diff
@@ -1368,3 +1373,111 @@ async def invoke_script(
                 ["foo", "bar"]
         """
         return await self._execute_script(script.get_hash(), keys, args)
+
+    @dataclass
+    class PubSubMsg:
+        """Describes incoming message
+
+        Attributes:
+            message (str): Incoming message.
+            channel (str): Name of an channel that triggered the message.
+            pattern (Optional[str]): Pattern that triggered the message.
+            context (Optional[str]): User-provided context for this subscription.
+        """
+
+        message: str
+        channel: str
+        pattern: Optional[str]
+        context: Optional[Any]
+
+    class ChannelModes(Enum):
+        """Describes subsciption modes."""
+
+        Exact = 1
+        """ Use exact channel names """
+        Pattern = 2
+        """ Use channel name patterns """
+        Sharded = 3
+        """ Use sharded PUBSUB. See https://redis.io/docs/interact/pubsub/#sharded-pubsub for more details """
+
+    async def subscribe(
+        self,
+        channels_or_patterns: Set[str],
+        channel_mode: CoreCommands.ChannelModes,
+        callback: Callable[[CoreCommands.PubSubMsg], None],
+        context: Optional[Any],
+    ) -> None:
+        """
+        Add specific channels or set of channels defined by a pattern to active subscriptions.
```

Comment:
`specific channels or set of channels` - what does this mean? what's the difference between specific channels and a set of channels?

Reply:
read the whole sentence - "...or set of channels defined by a pattern"

Reply:
rephrased - "Add specific channels or channels patterns to active subscriptions."
Done